public/app/plugins/datasource/mssql/partials/query.editor.html

```diff
@@ -10,7 +10,7 @@
     <div class="gf-form">
 			<label class="gf-form-label query-keyword">Format as</label>
 			<div class="gf-form-select-wrapper">
-				<select class="gf-form-input gf-size-auto" ng-model="ctrl.target.format" ng-options="f.value as f.text for f in ctrl.formats" ng-change="ctrl.refresh()"></select>
+				<select class="gf-form-input gf-size-auto" ng-model="ctrl.target.format" ng-options="f.value as f.text for f in ctrl.formats" ng-change="ctrl.refresh()" aria-label="Format as"></select>
```

Comment:
I think it would make sense to use the label element above by adding an id attribute to the select element and a for attribute to the label.

Reply:
You're right. I was trying to work around the issue of having multiple query editors. I've changed to use a dynamic ID instead.